docs/netserver/config/security.md

```diff
@@ -73,16 +83,34 @@ It is important to know that if your system is using a server cluster these valu
 
 Sentry configuration options.
 
-| Name | Description |
-|---|---|
-| Ignore | Ignore the Sentry mechanism, everything will be allowed.<br>Default: false |
+| Name | Description | Default |
+|---|---|---|
+| Ignore | Ignore the [Sentry mechanism][2], everything will be allowed. | false |
 
 ## SoPasswordCredentialPlugin
 
 SuperOffice password credentials plugin options.
 
+| Name | Description | Default |
+|---|---|---|
+| DisableUseExternalAssociate | Disables SuperOffice password for External associates. | false |
+| DisableUseInternalAssociate | Disables SuperOffice-passwords for Internal associates (normal users). | false |
+| DisableUseSystemAssociate | Disables SuperOffice password for System associates. | false |
+| PriorityInternalAssociate | Priority of the plugin for internal associates. | |
+
+## Authentication
```

Comment:
Where does this come from`? It does not appear anywhere I search in Security section.

Reply:
From https://docs.superoffice.com/api/reference/netserver/core/SuperOffice.Configuration.ConfigFile.Security.SoPasswordCredentialPlugin.html